composer.json

```diff
@@ -18,29 +18,31 @@
     "minimum-stability": "dev",
     "require": {
         "php": "^7.2",
+        "phpunit/phpunit": "^8.0",
         "psr/log": "^1.1",
         "psr/event-dispatcher": "^1.0",
+        "yiisoft/aliases": "^3.0@dev",
         "yiisoft/arrays": "^3.0@dev",
+        "yiisoft/cache": "^3.0@dev",
+        "yiisoft/di": "^3.0@dev",
+        "yiisoft/event-dispatcher": "^3.0@dev",
```

Comment:
Should depend on interfaces:

```json
"psr/event-dispatcher": "1.0.0",
"psr/event-dispatcher-implementation": "1.0.0"
```

If concrete implementation is needed for tests, move to `require-dev`.